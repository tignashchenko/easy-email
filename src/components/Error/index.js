import React from 'react';

const Error = () => <h1>Page not found (404 error)</h1>;

export default Error;
