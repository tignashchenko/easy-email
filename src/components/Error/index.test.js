import React from 'react';
import dom from 'react-test-renderer';
import Error from './';

const renderTree = dom.create(<Error />).toJSON();

test('Error component should correspond to its snapshot counterpart', () => {
    expect(renderTree).toMatchSnapshot();
});
