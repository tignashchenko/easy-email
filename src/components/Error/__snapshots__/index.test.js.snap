// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component should correspond to its snapshot counterpart 1`] = `
<h1>
  Page not found (404 error)
</h1>
`;
